index.html

```diff
@@ -273,30 +273,35 @@ <h2 class="categorie-title">
 </main>
 <footer class="footer-container" id="footer-container-id">
   <div class="footer-form">
-    <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
-    <form action="https://formspree.io/f/xzbovlzk" method="post">
-      <ul>
-        <li>
-          <input type="text" name="Name" placeholder="Full name" required maxlength="30">
-        </li>
-        <li>
-          <input type="email" name="Email" placeholder="Email address"  required>
-        </li>
-        <li>
-          <textarea name="Message" id="msg" maxlength="500" required >Enter text here</textarea>
-        </li>
-        <li class="submit-button">
-          <button type="submit">
-              Get in touch
-          </button><br>
-          <small></small>
-      </li>
-      </ul>
-    </form>
+      <h3 class="footer-title">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h3>
+      <form action="https://formspree.io/f/mrgjzokg" method="post" class="form-input">
+          <ul class="form-fields">
+              <li class="input-name">
+                  <input type="text" name="name" id="name" maxlength="30" placeholder="Full name" required>
 
-    
+              </li>
+              <li class="input-full-name">
+                  <input type="text" name="first_name" id="first_name" maxlength="30" placeholder="First name" required>
+                  <input type="text" name="last_name" id="last_name" maxlength="30" placeholder="Last name" required>
+              </li>
```

Comment:
- [ ] There should be one input for Full name (just like the mobile version) and this should take the space of both First Name and Last Name inputs (shown in the desktop version).